Example/package.json

```diff
@@ -3,21 +3,22 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "start": "metro-with-symlinks start",
     "test": "jest",
-    "postinstall": "rm -rf node_modules/react-native-reanimated/{.git,node_modules,Example}"
```

Comment:
I used this with `yarn link` and it deleted my git repo 😢 , but we can discuss on another PR.